.github/ISSUE_TEMPLATE/--bug-report.md

```diff
@@ -24,7 +24,7 @@ A clear and concise description of what the bug is.
 **To Reproduce**
 Steps to reproduce the behavior:
 
-[Include code or an example repository that can easily be set up]
+[Please provide an example repository that can be easily set up.  Otherwise the issue will get flagged as **Reproducible Example Missing** and ignored until such an example is provided]
```

Comment:
I don't think ignored is the right word here. Given the context of the issue we would still provide help and a lot of issues can be solved without having a reproducible example.

Could we rephrase the last sentence and avoid to say `ignored`?